python/py-plyvel/Portfile

```diff
@@ -4,35 +4,28 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-plyvel
-version             1.1.0
+version             1.5.0
 revision            0
 
 categories-append   devel
 license             BSD
 
-python.versions     37
+python.versions     27 38 39 310 311
```

Comment:
can we please not add PY27?

Reply:
@catap ?

Reply:
@reneeotten it is required by john-jumbo port.

see: https://github.com/macports/macports-ports/pull/20299

right now such dependencies simple doesn’t exist but it is a missed one 🥲

Reply:
The good news, the next release of `john-jumbo` will eliminate that.